docs/guides/ti/matter_cc2674p10_migration.md

```diff
@@ -0,0 +1,98 @@
+# Running Matter Examples on the TI SimpleLink CC2674P10 and CC2674R10
+
+The existing Matter project examples are based on LP_EM_CC1354P10_6. If using 
+the CC2674P10, the following migration steps are required. Developers are 
+strongly encouraged to start with a `cc13x4_26x4` example and migrate the 
+project accordingly. Example projects can be found in the following location: 
+`matter/examples/[application]/cc13x4_26x4`
+
+## Dependencies
+
+The following must be installed on your system before proceeding:
+
+* [SysConfig](https://www.ti.com/tool/SYSCONFIG) v1.16.2 or later
+* [SIMPLELINK-LOWPOWER-F2-SDK](https://www.ti.com/tool/SIMPLELINK-LOWPOWER-SDK) v7.10.01.24
+
+## Matter source code changes
+
+The following are changes to the Matter source code files which should be 
+applied to convert a `matter/examples/[application]/cc13x4_26x4` project to the
+CC2674P10 device
+
+* `examples/[application]/cc13x4_26x4/args.gni`, modify/add the following 
+  defines for the CC2674
+  - `ti_simplelink_board = CC2674`
+  - `ti_simplelink_device = CC2674P10RGZ`
+* `third_party/ti_simplelink_sdk/repo_cc13xx_cc26xx`, replace this folder 
+  contents with the 7.10.01.24 version from 
+  [TI's downloads page](https://www.ti.com/tool/download/SIMPLELINK-LOWPOWER-F2-SDK/7.10.01.24) which is required to add support SDK for the CC2674P10 device.

```

Comment:
Are there plans to update the submodule, in order to skip this step?

Reply:
@ReneJosefsen this is tentatively planned for next week.